hms-app/src/components/buttons/ParticipateButton.tsx

```diff
@@ -181,18 +182,21 @@ export default function ParticipateButton(props: IProps) {
   }, [ideaData])
 
   return (
-    <Button
-      disabled={buttonIsDisabled}
-      onClick={
-        participantCheck ? removeThisIdeaParticipant : addThisIdeaParticipant
-      }
-      style={{
-        backgroundColor: participantCheck
-          ? LEAVE_BUTTON_COLOR
-          : JOIN_BUTTON_COLOR,
-      }}
-    >
-      {participantCheck ? 'Leave Idea' : 'Join Idea'}
-    </Button>
+    <>
+      <Group>
+        <Text className={classes.boldText}>
+          {participantCheck ? 'Leave' : 'Participate'}
+        </Text>
+        <Switch
```

Comment:
maybe we can use this kind of switch, so that we can always use the word "participate"